tests/scripts/save_policy_to_safetensors.py

```diff
@@ -108,15 +108,15 @@ def save_policy_to_safetensors(output_dir, env_name, policy_name, extra_override
 
 if __name__ == "__main__":
     env_policies = [
-        ("xarm", "tdmpc", []),
-        (
-            "pusht",
-            "diffusion",
-            ["policy.n_action_steps=8", "policy.num_inference_steps=10", "policy.down_dims=[128, 256, 512]"],
-        ),
-        ("aloha", "act", ["policy.n_action_steps=10"]),
-        ("dora_aloha_real", "act_real", ["policy.n_action_steps=10"]),
-        ("dora_aloha_real", "act_real_no_state", ["policy.n_action_steps=10"]),
+        # ("xarm", "tdmpc", []),
+        # (
+        #     "pusht",
+        #     "diffusion",
+        #     ["policy.n_action_steps=8", "policy.num_inference_steps=10", "policy.down_dims=[128, 256, 512]"],
+        # ),
+        ("aloha", "act_1000_actions", []),
```

Comment:
Maybe @aliberts you have an opinion on this question around tests and config file.

See my `How it was tested` section in this PR description.

Reply:
Oh sorry, I meant also to change to using just the act.json file. Also happy to hear what @aliberts says.

Reply:
oh yes this also doesn't work because the safetensor files overide themselves (cf my comment in the PR description. maybe a bit short) unless we either:
- decide we test always in this 1000 action setup for all policy tests
- decide to update the way we name safetensor files

Reply:
> unless we either:
> * decide we test always in this 1000 action setup for the other ACT policy tests as well
> * decide to update the way we name safetensor files

IMO both are fine (the naming trick you went with is okay, although it adds artifacts they're relatively small). I was concerned with speed too but from what I'm seeing in CI tests it doesn't increase tests duration (cf [this branch](https://github.com/huggingface/lerobot/actions/runs/9578846846/job/26409975290?pr=279#step:7:258) vs [main](https://github.com/huggingface/lerobot/actions/runs/9565050029/job/26367137322#step:7:255))

Long-term though, I think it'll be much better to have more fine-grained tests and tests individual components of the policies and do away with these artifacts, similar to what's done in transformers. My motivations for this are:
- Whenever this test breaks, it doesn't tell us *which* part broke it.
- Having this many tests artifacts as we do right now is really not great for cloning the repo.

For now it's okay because this granularity allows us to iterate faster while still having guardrails. Happy to hear your opinions on this as well.